src/amo/pages/Addon/index.js

```diff
@@ -489,7 +489,7 @@ export class AddonBase extends React.Component {
                     <a href="%(newLocation)s">Browse add-ons for Firefox on desktop</a>.`,
                 )}
                 fixFenixLinkMessage={i18n.gettext(
-                  `Not available on Firefox for Android. You can use this add-on with Firefox for Desktop. 
+                  `Not available on Firefox for Android. You can use this add-on with Firefox for Desktop.
```

Comment:
/me wonders how a trailing space got through linting 🤔

Reply:
This is a multi-line string so it is valid. My vim config removes such spaces, though.